eth-custodian/scripts/eth_deploy_proxy.js

```diff
@@ -0,0 +1,39 @@
+const hre = require('hardhat');
+
+const ethereumConfig = require('./json/ethereum-config.json');
+
+async function main() {
+    const [deployerAccount] = await hre.ethers.getSigners();
+
+    console.log(`Deploying proxy with the account: ${deployerAccount.address}`);
+
+    const ethCustodianProxyContractFactory = await hre.ethers.getContractFactory('EthCustodianProxy');
+    const proxy = await hre.upgrades.deployProxy(
+        ethCustodianProxyContractFactory,
+        [ethereumConfig.ethConnectorAddress]
+    );
+
+    await proxy.deployed();
+
+    console.log("EthCustodianProxy deployed to:", proxy.address);
+
+    const ethCustodianContractFactory = await hre.ethers.getContractFactory('EthCustodian');
+    const ethCustodian = ethCustodianContractFactory.attach(ethereumConfig.ethConnectorAddress);
+
+    const nominateTx = await ethCustodian.nominateAdmin(proxy.address);
```

Comment:
The current admin is a multisig, so this will not work.

Reply:
Removed 16da39ba2ea0ff777a6a68f1a3f695bcd2819178